### PR TITLE
Move and slightly improve simple tests

### PR DESCRIPTION
**Pull request by obnoxxx**
This is in preparation of introducing the complex vagrant-based tests.

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/235)

### CODE REVIEW

File: tests/simple/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+but are syntax-checks or unit-tests, or use mocking or stubbing
+to test specific aspects.
+
+## Prerequisites
+
+The yaml tests require the 'yamllint' program.
+Install it with e.g.
+
+* `dnf install yamllint`, or
+* `pip install yammling`
```
Comment (jarrpa) [inline tests/simple/README.md]: yammling?

File: tests/simple/yaml/test_syntax.sh [hunk 2]
```diff
@@ -36,6 +36,14 @@ check_invalid_yaml () {
 	return 0
 }
 
+if ! which yamllint >/dev/null 2>&1 ; then
+	subunit_start_test "yaml syntax tests"
```
Comment (jarrpa) [inline tests/simple/yaml/test_syntax.sh]: Do we need this for the timestamp?

File: tests/simple/README.md (hunk 1 above)
Comment (obnoxxx) [inline tests/simple/README.md]: fixed

File: tests/simple/yaml/test_syntax.sh (hunk 2 above)
Comment (obnoxxx) [inline tests/simple/yaml/test_syntax.sh]: yes

**Comment by obnoxxx** [comment]
@jarrpa addressed your comments.

File: tests/simple/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+but are syntax-checks or unit-tests, or use mocking or stubbing
+to test specific aspects.
+
+## Prerequisites
+
+The yaml tests require the 'yamllint' program.
+Install it with e.g.
+
+* `dnf install yamllint`, or
+* `pip install yammlint`
```
Comment (jarrpa) [inline tests/simple/README.md]: `yammlint` -> `yamllint` :)

Comment (obnoxxx) [inline tests/simple/README.md]: gosh .... done

**Comment by obnoxxx** [comment]
@jarrpa : 
* fixed the typo in the readme
* fixed the commit msg for the makefiles
* squashed the "add central test readme" commit into move simple tests to their own dir commit
* refusing to squash the "fix typo" and "explain simple tests" readme commits, since they are unrelated.
Hence I think I have addressed all your requests, and answered to the "maybe" part... :-)

**Comment by jarrpa** [comment]
LGTM

**Pull request merged**